llama-index-integrations/llms/llama-index-llms-maritalk/README.md

```diff
@@ -1 +1,19 @@
 # LlamaIndex Llms Integration: Maritalk
+
+MariTalk is an assistant developed by the Brazilian company [Maritaca AI](https://www.maritaca.ai). MariTalk is based on language models that have been specially trained to understand Portuguese well.
+
+## Installation
+
+First, install the Llama Index library (and all its dependencies) using the following command:
+
+```
+$pip install llama-index llama-index-llms-maritalk
+```
+
+## API Key
+
+You will need and API key that can be obtained from chat.maritaca.ai (\"Chaves da API\" section).
+
+## Examples
+
+Examples of usage are presented in the `maritalk.ipynb` file within this project.
```

Comment:
Since this shows up on llamahub.ai, we should directly link to the example (using the docs link I think, https://docs.llamaindex.ai/en/stable/examples/llm/maritalk/)